src/test/java/org/minions/devfund/katerinaanzoleaga/battleship/ShipTest.java

```diff
@@ -0,0 +1,280 @@
+package org.minions.devfund.katerinaanzoleaga.battleship;
+
+import junitparams.Parameters;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+//import org.junit.experimental.categories.Category;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)